tests/test_ion_diffusion_and_pumps.py

```diff
@@ -0,0 +1,474 @@
+# This file is part of Jaxley, a differentiable neuroscience simulator. Jaxley is
```

Comment:
Having a regression test case for pumps / diffusion would be great I think.

Reply:
A test that asserts against some hardcoded results, i.e. like in `jaxley_identical`, would also be a good tripwire for the future I think.

Reply:
As talked about, storing the voltages from the NEURON test somewhere, would forgoe the need to compile the channel models and we could get rid of the `additional_neuron_tests` flag and related alltogether.